vault/mount.go

```diff
@@ -294,6 +295,21 @@ func (t *MountTable) findByBackendUUID(ctx context.Context, backendUUID string)
 	return nil, nil
 }
 
+func (t *MountTable) findByAccessor(ctx context.Context, accessor string) (*MountEntry, error) {
+	n := len(t.Entries)
+	ns, err := namespace.FromContext(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	for i := 0; i < n; i++ {
+		if entry := t.Entries[i]; entry.Accessor == accessor && entry.Namespace().ID == ns.ID {
+			return entry, nil
+		}
+	}
```

Comment:
Might want to make a map/cache of this in case there are a lot of auth mounts.

Reply:
Kinda surprised we don't need a read lock for the iteration across t.Entries to prevent data races

Reply:
I believe this works because MountTables are replaced wholesale every time they're changed, they're not modified in place.